src/main/java/org/sobotics/guttenberg/utils/PostUtils.java

```diff
@@ -248,6 +248,8 @@ public static String storeReport(PostMatch match) throws IOException {
 		                "score", ""+match.getTotalScore(),
 		                "reasons", match.getCopyPastorReasonString());
 		
+		System.out.println(match.getCopyPastorReasonString());
```

Comment:
Use LOGGER instead of SOPln

Reply:
Logger didn't work. That's why I did the second commit ;-) :-D

Reply:
That's because you are using the wrong class. Line 29 should be `PostUtils.class` :-P